packages/docs/operations-manual/key-management/key-rotation.md

```diff
@@ -18,13 +23,22 @@ Because the Validator signer key is constantly in use to sign consensus messages
   # With $SIGNER_TO_AUTHORIZE as the new validator signer:
 
   # On the new validator node which contains the new $SIGNER_TO_AUTHORIZE key
-  docker run -v $PWD:/root/.celo --rm -it $CELO_IMAGE --nousb account proof-of-possession $SIGNER_TO_AUTHORIZE $LOCKED_GOLD_ACCOUNT
-  docker run -v $PWD:/root/.celo --rm -it $CELO_IMAGE --nousb account proof-of-possession $SIGNER_TO_AUTHORIZE $LOCKED_GOLD_ACCOUNT --bls
-
-  # On a node containing your Locked Gold Account key.
-  celocli account:authorize --from $LOCKED_GOLD_ACCOUNT --role validator --signer $SIGNER_TO_AUTHORIZE --signature 0x$SIGNER_PROOF_OF_POSSESSION --blsKey $BLS_PUBLIC_KEY --blsPop $BLS_PROOF_OF_POSSESSION
+  docker run -v $PWD:/root/.celo --rm -it $CELO_IMAGE --nousb account proof-of-possession $SIGNER_TO_AUTHORIZE $VALIDATOR_ACCOUNT_ADDRESS
```

Comment:
VALIDATOR_LOCKED_GOLD_ACCOUNT_ADDRESS?

Reply:
Because the Locked Gold account and the Validator Account use the address of the key or the contract, I didn't feel like the distinction was necessary. Is there a particular reason you suggest keeping locked gold in the name.